File: PMU/lab4/app/src/main/java/ru/lyovkin/lab3/MainActivity.java
package ru.lyovkin.lab4;

import androidx.appcompat.app.AppCompatActivity;
import android.opengl.GLSurfaceView;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        GLSurfaceView GLView = new GLSurfaceView(this);
        GLView.setEGLContextClientVersion(2);
        GLView.setRenderer(new MyRenderEngine());
        setContentView(GLView);
    }
}

File: PMU/lab3/app/src/main/java/ru/lyovkin/lab3/MainActivity.kt
package ru.lyovkin.lab3

import android.opengl.GLSurfaceView
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import ru.lyovkin.lab3.entity.SolarSystem

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val view = GLSurfaceView(this)
        view.setRenderer(SolarSystem(this))
        setContentView(view)
    }
}